docs/intro/go.md

```diff
@@ -0,0 +1,107 @@
+---
+order: 2
+description: Install the Go programming language
+---
+
+# Install Go
+
+In this tutorial, you will install the Go programming language (golang) on your local computer. Follow the instructions for your operating system:
+
+## macOS
+
+* Download the latest MacOS installer package from <https://golang.org/dl/>.
+* Open the downloaded package and follow the prompts through to completion.
+* By default, the package installs the Go distribution to `/usr/local/go`, however it is always best to define the path explicitly:
+
+    1. Open or create a `~/.bash_profile` file with your favorite command-line text editor.
```

Comment:
```suggestion
```

This might be different for everyone depending on the shell you're using.

Reply:
@ilgooz what about using `~/.bash_profile` / `~/.profile`?

Reply:
@ilgooz what about using `~/.profile`?

Reply:
Bdw, Isn't `~/.bash_profile` is available by default? :)

Reply:
yes, we can do the .profile way but we should also document that it needs to be sourced.

no, bash profile isn't available for different shells by default.